### PR TITLE
less than OR equal to.

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

File: style_editor.html [hunk 1]
```diff
@@ -9525,7 +9525,7 @@
 /* Variant Slider functions */
 
 function updateVariantValue(newValue) {
-    if (newValue < 0) {
```
Comment (profezzorn) [inline style_editor.html:9528]: Under what circumstances would this make a difference?

Comment (NoSloppy) [reply style_editor.html:9528]: I have a safeguardInputs() function coming that will prevent NaNs.
It autofills in a zero in the case where the user deletes the value completely.
WavLen will fill in a 1 as a minimum.
This one character fix for now allows the autofilled zero to be taken into account.

**Comment by NoSloppy** [comment]
add with later PR instead

**Comment by NoSloppy** [comment]
add with later PR instead

**Pull request closed without merging**